src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -83,15 +83,25 @@
     (text "hello 1234").low-cased
     $.equal-to "hello 1234"
 
-[] > text-upper-case
+[] > text-upper-case-1
   assert-that > @
-    (text "HeLlO wOrLd!").up-cased
-    $.equal-to "HELLO WORLD!"
+    (text "HeLlO !").up-cased
+    $.equal-to "HELLO !"
 
 [] > text-upper-case-2
   assert-that > @
-    (text "hello 123").up-cased
-    $.equal-to "HELLO 123"
+    (text "hel 3").up-cased
```

Comment:
same question here, why test content was changed?

Reply:
I did it to reduce time of execution.

Reply:
removing two symbols reduces time of execution? doesn't it mean that the method is _really_ slow?

Reply:
@OlesiaSub yes it is slow but it's not about the algorithm. At the moment the language is slow and the same algoritm would be much faster on java.

Reply:
@OlesiaSub We can try to use `thread`s here to calculate parts of the string in parallel when the are released

Reply:
@OlesiaSub We also can try to use `heap` but it similar to usage of mutability

Reply:
@levBagryansky I think it's ok if it is slow because of the language. You can keep in mind the idea of using threads though and maybe implement it some time later.

Reply:
@OlesiaSub Please also check #89. There is implementation by `heap`.